packages/frontend/src/components/wallet/NFTs.js

```diff
@@ -57,14 +57,14 @@ const StyledContainer = styled.div`
     }
 `;
 
-const NFTs = ({ tokens }) => {
+const NFTs = ({ tokens, accountId }) => {
     if (tokens.length) {
         return (
             <StyledContainer>
                 {tokens.map((tokenDetails) => (
                     <NFTBox
                         key={tokenDetails.contractName}
-                        tokenDetails={tokenDetails}
+                        tokenDetails={Object.assign({ ownerId: accountId }, tokenDetails)}
```

Comment:
`ownerId` should already be available in the `tokenDetails`, given that `tokenDetails` includes the metadata from on-chain RPC call results..  Why are we setting `ownerId` to `accountId` here?